feature-libs/product-configurator/rulebased/occ/variant/variant-configurator-occ.adapter.ts

```diff
@@ -230,22 +230,23 @@ export class VariantConfiguratorOccAdapter
         urlParams: {
           configId: configuration.configId,
         },
+        queryParams: { groupId: configuration.interactionState.currentGroup },
       }
     );
 
     return this.http.get(url).pipe(
-      this.converterService.pipeable(
-        VARIANT_CONFIGURATOR_PRICE_SUMMARY_NORMALIZER
-      ),
-      map((pricingResult) => {
+      this.converterService.pipeable(VARIANT_CONFIGURATOR_PRICE_NORMALIZER),
+      map((configResult) => {
         const result: Configurator.Configuration = {
           ...configuration,
-          priceSummary: pricingResult,
+          priceSummary: configResult.priceSummary,
+          priceSupplements: configResult.priceSupplements,
```

Comment:
The method is called readPriceSummary, but the priceSupplements are set here as well. This is kind of misleading. We cannot rename the method can we?

Reply:
You are right! But as you suppose we think the effort to rename the method is too high. We want to keep 'readPriceSummary' even if a bit misleading now. We can think of migrating it in a future major release